libs/pageserver_api/src/key.rs

```diff
@@ -140,3 +140,7 @@ impl Key {
         })
     }
 }
+
+pub fn is_rel_block_key(key: &Key) -> bool {
```

Comment:
To avoid having to reason about the existing use, I've added an explicit exclusion of the relation size key in this function, and also a comment explaining the logic (92d7a82e0)